xds/src/test/java/io/grpc/xds/PriorityLoadBalancerProviderTest.java

```diff
@@ -36,6 +36,7 @@
 public class PriorityLoadBalancerProviderTest {
   @Rule public final ExpectedException thrown = ExpectedException.none();
 
+  @SuppressWarnings("ExpectedExceptionChecker")
```

Comment:
Without this annotation I don't see any lint warnings. Any special flags I need to use?

Reply:
In google3, there is.